### DISCUSSION TITLE
how to solve FDM on second-order partial differential equations in 3D

### DISCUSSION THREAD

**Discussion by mumu-1005** [zwicker-group/py-pde#146] [Q&A]
u_t = a^2 (u_xx + u_yy + u_zz)

https://en.wikipedia.org/wiki/Heat_equation

**Comment by david-zwicker** [comment]
The heat equation is equivalent to the diffusion equation, so you can look at these examples.

**Reply by mumu-1005** [reply]
Can I use the py-pde to solve laplace's equation in 3d?

**Reply by mumu-1005** [reply]
a independent system including two cubic coupled by one cylindrical

**Reply by david-zwicker** [reply]
It's not clear to me what you're asking. Could you give more details?

**Reply by mumu-1005** [reply]
the system like this
![image](https://user-images.githubusercontent.com/17617655/135266072-9ba6d5d6-5afe-4c33-b86a-657768bd631c.png)

**Reply by david-zwicker** [reply]
I'm still not sure which equation you're trying to solve, but `py-pde` is not designed to handle complex geometries. You might want to look into finite element or finite volume methods.

**Reply by mumu-1005** [reply]
Get it! Thank you very much!
